services/apps/alcs/src/alcs/notice-of-intent/notice-of-intent-submission-status/notice-of-intent-submission-status.controller.ts

```diff
@@ -0,0 +1,45 @@
+import { Mapper } from '@automapper/core';
+import { InjectMapper } from '@automapper/nestjs';
+import { Controller, Get, Param } from '@nestjs/common';
+import { ANY_AUTH_ROLE } from '../../../common/authorization/roles';
+import { UserRoles } from '../../../common/authorization/roles.decorator';
+import { NoticeOfIntentSubmissionToSubmissionStatusDto } from './notice-of-intent-status.dto';
+import { NoticeOfIntentSubmissionToSubmissionStatus } from './notice-of-intent-status.entity';
+import { NoticeOfIntentSubmissionStatusService } from './notice-of-intent-submission-status.service';
+
+@Controller('notice-of-intent-submission-status')
+@UserRoles(...ANY_AUTH_ROLE)
+export class NoticeOfIntentSubmissionStatusController {
+  constructor(
+    private applicationSubmissionStatusService: NoticeOfIntentSubmissionStatusService,
```

Comment:
Rename `applicationSubmissionStatusService` to something noi-related

Reply:
Done!